tst/testbugfix/2018-09-13-MTC.tst

```diff
@@ -1,4 +1,6 @@
 #MTC memory request/drop issue , Github PR#2812
+# was ``reported'' in forum email
```

Comment:
Why in quotes? Yes, the report was incomplete, but it was a genuine report, and we were able to confirm the problem...

Reply:
As it was incomplete and the request to submit further detail was unanswered.

Reply:
I mean, how can I find a report on the "old" MTC bug, which prompted the rewrite?

Reply:
@dimpase 
The bug in the old MTC was #302, the change of the MTC #843